.github/workflows/spack.yml

```diff
@@ -14,17 +14,53 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout Spack
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         repository: spack/spack
         path: ./spack
     - name: Checkout mochi-spack-packages
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         repository: mochi-hpc/mochi-spack-packages
         path: ./mochi-spack-packages
-    - name: Run a multi-line script
+    - name: Install APT Dependencies
+      run: |
+        sudo apt-get update
+        sudo apt-get install -q -y autoconf
+        sudo apt-get install -q -y automake
+        sudo apt-get install -q -y cmake
+        sudo apt-get install -q -y libtool
+        sudo apt-get install -q -y libtool-bin
+        sudo apt-get install -q -y mpich
+        sudo apt-get install -q -y zlib1g-dev
+        sudo apt-get install -q -y diffutils
+        sudo apt-get install -q -y libdb-dev         
+        sudo apt-get install -q -y libedit-dev
+        sudo apt-get install -q -y libncurses6
+        sudo apt-get install -q -y libxml2-dev
+        sudo apt-get install -q -y openssl        
+    - name: Run spack install mobject
       run: |
         . ./spack/share/spack/setup-env.sh
+        spack compiler find
+        spack compilers
+        spack external find autoconf
+        spack external find automake
+        spack external find berkeley-db
+        spack external find boost
+        spack external find cmake
+        spack external find diffutils
+        spack external find libedit
+        spack external find libtool
+        spack external find libiconv
+        spack external find libxml2
+        spack external find m4
+        spack external find mpich
+        spack external find ncurses
+        spack external find openssl
+        spack external find perl
+        spack external find pkgconf
+        spack external find zlib
+        spack external find xz        
         spack repo add ./mochi-spack-packages
-        spack install --test root mobject
```

Comment:
@mdorier can you confirm this change is okay? The `--test root` option does seem to lead to spack failures here (they aren't very verbose, you can probably seem them in previous workflow failures like https://github.com/mochi-hpc/mobject/actions/runs/3577863276/jobs/6017346326).

I'm not sure what recent change would lead to those failures, but dropping the option does allow tests to pass on this PR.